needles/gnome_software_encyclopedia_details_uninstalled.json

```diff
@@ -0,0 +1,32 @@
+{
+  "tags": [
+    "gnome_software_details",
+    "gnome_software_encyclopedia_details",
+    "gnome_software_encyclopedia_details_uninstalled"
+  ],
+  "area": [
+    {
+      "xpos": 0,
+      "ypos": 0,
+      "width": 1024,
+      "height": 768,
+      "type": "match"
+    },
+    {
+      "": "Screenshots often change or are not downloaded",
```

Comment:
Well, they're meant to be pre-cached in the image.

Reply:
Hmm, I recall that they were grey for me on a few boots, but I assumed that was normal and didn’t investigate further. Is it worth filing a follow-up issue about this?

In any case I think it is appropriate to have an exclusion area here, since we’re not testing the screenshots, and they will eventually change over time.

Reply:
>  Is it worth filing a follow-up issue about this?

Probably, if you see it again.